Source/Charts/Charts/PieChartView.swift

```diff
@@ -597,6 +599,26 @@ open class PieChartView: PieRadarChartViewBase
             return drawEntryLabelsEnabled
         }
     }
+
+    // Set this to true to draw the value text in entry labels into the pie slices
+    open var drawValueTextEnabled: Bool
```

Comment:
We are no longer using this pattern for `Bool`s. We are just using one with the proper name `isDrawValueTextEnabled`